### ISSUE TITLE
"Unrecognized command or argument" if output file has spaces in the path and using DsComTypeLibraryEmbedAfterBuild=true

### ISSUE THREAD

**Issue by icnocop** [dspace-group/dscom#287]
Hi.

Thank you for dscom.

When trying to use the NuGet package `dSPACE.Runtime.InteropServices.BuildTasks` to generate a type library (TLB) from my .net assembly, I get the following error:

```
...
        Using arguments '       tlbexport "C:\dev\My Solution\MyProject\bin\Debug\net8.0\MyProject.dll" --embed C:\dev\My Solution\MyProject\obj\Debug\net8.0\MyProject.comhost.dll     --out "C:\dev\My Solution\MyProject\bin\Debug\net8.0\\MyProject.tlb"' to call dscom
...
        Unrecognized command or argument 'Solution\MyProject\obj\Debug\net8.0\MyProject.comhost.dll'.
...
      Done executing task "Exec" -- FAILED.
    Done building target "DsComExportTypeLibraryAfterBuild" in project "MyProject.csproj" -- FAILED.
```

This happens because there are spaces in the comhost file path.

I expected the comhost file path to be surrounded with double quotes when passed to the `--embed` command line parameter.

Using version 1.11.0 and the following MSBuild property values:

```
    <_DsComForceToolUsage>true</_DsComForceToolUsage>
    <DsComTypeLibraryEmbedAfterBuild>true</DsComTypeLibraryEmbedAfterBuild>
```

Thank you.

**Comment by SOsterbrink** [comment]
I'll try to take a closer look at it and fix it tomorrow. But that seems to be a bug in our command line parameter handling.

**Comment by SOsterbrink** [comment]
The bug-fix was smaller than expected: v1.11.1 should fix the problem and should be available soon

**Comment by marklechtermann** [comment]
[https://github.com/dspace-group/dscom/releases/tag/v1.11.1](https://github.com/dspace-group/dscom/releases/tag/v1.11.1)
